packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -404,3 +406,14 @@ export function getStylesConfig(wco: WebpackConfigOptions): Configuration {
     plugins: extraPlugins,
   };
 }
+
+function getSassImplementation(): SassWorkerImplementation | typeof import('sass') {
+  const { webcontainer } = process.versions as unknown as Record<string, unknown>;
+
+  // When `webcontainer` is a string it means that we are running in a StackBlitz webcontainer.
+  // `SassWorkerImplementation` uses `receiveMessageOnPort` Node.js `worker_thread` API to ensure sync behavior which is ~2x faster.
+  // However, it is non trivial to support this in a webcontainer and while slower we choose to use `dart-sass`
+  // which in Webpack uses the slower async path.
+  // We should periodically check with StackBlitz folks (Mark Whitfeld / Dominic Elm) to determine if this workaround is still needed.
+  return typeof webcontainer === 'string' ? require('sass') : new SassWorkerImplementation();
```

Comment:
Nit: Do we care that `webcontainer` is a `string`? Would it be safer/cleaner to just check if the property exists / is truthy?

Reply:
I think we can check if it’s truthy. But  stackblitz folks suggested this way.

@markwhitfeld thoughts?

Reply:
The value will be a commit sha and it should be a string yes. I think either works.